src/map/battle.cpp

```diff
@@ -8139,6 +8139,13 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 
 	battle_do_reflect(BF_WEAPON,&wd, src, target, skill_id, skill_lv); //WIP [lighta]
 
+	if (std::shared_ptr<s_skill_db> skill = skill_db.find(skill_id);
+		sd != nullptr
+		&& !(wd.type == DMG_CRITICAL || wd.type == DMG_MULTI_HIT_CRITICAL)
+		&& !(skill != nullptr && skill->inf2[INF2_IGNORENORMALATKBONUS])) {
+		ATK_ADDRATE(wd.damage, wd.damage2, sd->bonus.normal_atk_rate);
```

Comment:
Is it correct to place this at the very end of the calculation? For example now reflect damage will not be affected by it. Also mastery damage bonuses will be increased by it.

Reply:
Maybe the bonus should be around https://github.com/rathena/rathena/blob/bdffa63956f7c70b2391aa280b514d58641181cf/src/map/battle.cpp#L8007

For reference https://ro.gnjoy.com/news/notice/View.asp?BBSMode=10001&seq=8115&curpage=1
```
> The increased physical damage per hit effect applies to regular physical attacks that do not trigger a critical hit, or to attacks that trigger a critical hit but do not inflict critical damage. 
> Does not apply to magic skills, fixed damage skills, skills that do not receive a melee/ranged physical damage increase effect, and some exceptional skills.
```